assignments/prototypes.js

```diff
@@ -15,6 +15,15 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject (createdAt, dimensions) {
+  this.createdAt = createdAt,
+  this.dimensions = dimensions,
```

Comment:
You'll want to handle this like: 
```
function GameObject (attributes) {
  this.createdAt = attributes.createdAt,
  this.dimensions = attributes.dimensions,
```
Otherwise, you'll run into issues later when creating objects that inherit this properties. Currently, ` console.log(mage.createdAt)` logs out an entire object, instead of just the date time.